src/bioregistry/data/bioregistry.json

```diff
@@ -88016,7 +88016,7 @@
       "namespaceEmbeddedInLui": false,
       "pattern": "^PIRSF\\d{6}$",
       "prefix": "pirsf",
-      "uri_format": "https://pir.georgetown.edu/cgi-bin/ipcSF?id=$1"
```

Comment:
One important request for change: generally, we shouldn't edit the entries within blocks representing content from other resources mapped to Bioregistry. `n2t` is one example, where in this block, we are simply propagating the URL provided by `n2t`. If we want to override the Bioregistry's prioritized URL, we should curate that as a `uri_format` key at the top level under the given resource's key. This will override any `uri_format` entries from mapped registries like `n2t`.